src/io/flutter/run/daemon/FlutterApp.java

```diff
@@ -194,6 +196,19 @@ public static FlutterApp fromProjectProcess(@NotNull Project project) {
     return null;
   }
 
+  @NotNull
+  public static List<FlutterApp> collectAllApps() {
+    List<FlutterApp> apps = Lists.newArrayList();
+    Project[] projects = ProjectManager.getInstance().getOpenProjects();
+    for (Project project : projects) {
+      FlutterApp app = fromProjectProcess(project);
```

Comment:
what is the project has multiple apps running? e.g. Android and iOS

Reply:
Each app has its own OS process, doesn't it?

Reply:
so my concern is I don't quite understand how this can get all FlutterApp objects as a Project could have multiple running apps at the same time

Reply:
probably the solution is to modify 
 public static FlutterApp fromProjectProcess(@NotNull Project project)

to
 public static List<FlutterApp> fromProjectProcess(@NotNull Project project) {